main.go

```diff
@@ -1841,3 +1862,61 @@ func exitIfError(msg string, err error) {
 		log.WithFields(log.Fields{"err": err}).Fatalf(msg)
 	}
 }
+
+func enableDevMode(device ios.DeviceEntry, enablePostRestart bool) {
```

Comment:
I wonder if this command should go to the module and not be in main.go
Could it be useful for others too? What do you think?

Reply:
Mmm I guess people would want to wrap it their way but probably it will be useful for it to be already there. I'll see if I can move it and leave it as cool :D

Reply:
maybe provide both? low level control and the high level one here

Reply:
I moved it completely to the amfi package, just calling it and getting errors in main.go 👌